enginetest/trigger_queries.go

```diff
@@ -1862,6 +1862,121 @@ end;`,
 			},
 		},
 	},
+	{
+		Name: "simple trigger with non-existent table in trigger body",
```

Comment:
How about any other resolved references that we would want to avoid erroring on until execution of the trigger/procedure/function instead of at creation time? 

I did a quick check with MySQL and non-existent functions referenced in the body of a trigger/procedure/etc will also process just fine, and then error out in execution. It would be good to think about any other references that we may want to skip resolution on.

Reply:
Ah that's good find, I'll add support for allowing non-existent procedures in create trigger as well, and check in with other unresolved references.